src/utils/date.ts

```diff
@@ -116,6 +118,29 @@ export const formatTimeRange = (date: string | Date) => {
   });
 };
 
+export const formatDateToSeconds = (date: string | Date) =>
```

Comment:
Don't use a new function/format

for the duration.Humanize()

+ Customize durations across the board:

Use d vs day,

Reply:
@moshloop is this what you had in mind?